packages/@webex/internal-plugin-dss/src/dss.ts

```diff
@@ -257,6 +258,27 @@ const DSS = WebexPlugin.extend({
       },
     });
   },
+
+  /**
+   * Search for information about places
+   * @param {Object} options
+   * @param {string} options.queryString A query string that will be transformed into a Directory search filter query. It is used to search the following fields: placeName, displayName.
+   * @param {number} options.resultSize The maximum number of results returned from each provider
+   * @returns {Promise} Resolves with an array of entities found
+   */
+  searchPlaces(options: SearchPlaceOptions) {
+    const {resultSize, queryString, isOnlySchedulableRooms} = options;
+
+    return this._request({
+      dataPath: 'directoryEntities',
+      resource: `/search/orgid/${this.webex.internal.device.orgId}/places`,
+      params: {
+        queryString,
+        resultSize,
+        isOnlySchedulableRooms,
+      },
+    });
+  },
```

Comment:
i would add a catch block log the error and throw the error from there

Reply:
add the catch block, please review it again. Thanks